pkg/packagekit/updates.jsx

```diff
@@ -724,6 +724,7 @@ class OsUpdates extends React.Component {
 
                                            if (exit === PK.Enum.EXIT_SUCCESS) {
                                                this.setState({ state: "updateSuccess", loadPercent: null });
+                                               this.loadHistory();
```

Comment:
I think this should also be done on failures or cancels. That state just applies to the current update, while the history applies to the previous one, so it should be independent.

Reply:
That would be a change in logic,  no?  Here I have just moved the call to `loadHistory` from rendering the updateSuccess state to setting the state.  `loadHistory` wasn't called earlier in the cancel or failure case.

However, `loadUpdates` calls `loadHistory`.  Is that maybe enough and we can remove the call to `loadHistory` altogether here?

Reply:
Good point,  I missed that before. This applies to the history shown after applying updates succeeded, which is the screen that recommends/offers to reboot. `loadUpdates()` shows the available updates, thus it is a different page, and thus unrelated to the former. So in summary, this looks good. Thanks!